src/components/document-list/_macro.njk

```diff
@@ -76,12 +77,11 @@
                                     {%- if document.metadata.file and document.metadata.file.fileType %}
 
                                         <li class="ons-document-list__item-attribute" aria-hidden="true">
-                                            {% set fileMetadataItems = [] %}
-
-                                            {% if document.metadata.file.fileType %}{% set fileMetadataItems = (fileMetadataItems.push(document.metadata.file.fileType), fileMetadataItems) %}{% endif %}
-                                            {% if document.metadata.file.fileSize %}{% set fileMetadataItems = (fileMetadataItems.push(document.metadata.file.fileSize), fileMetadataItems) %}{% endif %}
-                                            {% if document.metadata.file.filePages %}{% set fileMetadataItems = (fileMetadataItems.push(document.metadata.file.filePages), fileMetadataItems) %}{% endif %}
-
+                                            {% set fileMetadataItems = [
+                                                document.metadata.file.fileType if document.metadata.file.fileType, 
```

Comment:
```suggestion
                                                document.metadata.file.fileType if document.metadata.file.fileType,
```
This is also an issue in the other change too

Reply:
This is also on line 36

Reply:
done